scss/helpers/_colored-links.scss

```diff
@@ -1,4 +1,14 @@
 @each $color, $value in $theme-colors {
+  // --bs-link-color and --bs-link-color-hover variables
+  .links-#{$color} {
+    --#{$variable-prefix}link-color: #{$value};
+    @if $link-shade-percentage != 0 {
+      --#{$variable-prefix}link-hover-color: if(color-contrast($value) == $color-contrast-light, shade-color($value, $link-shade-percentage), tint-color($value, $link-shade-percentage));
+    } @else {
+      --#{$variable-prefix}link-hover-color: #{$value};
+    }
+  }
```

Comment:
This could be easier I think: simply declare your custom properties in `.link-#{$color}` above and drop this whole new block. Then using `.link-*` would impact every links inside. No need to add a pattern, IMHO.

What do you think about it?

Reply:
Thanks, this variant looks better.